go/database/mpt/verification_test.go

```diff
@@ -407,11 +409,93 @@ func TestVerification_ValueNodeHashModificationIsDetected(t *testing.T) {
 			node.hash[12]++
 		})
 
-		if err := VerifyFileForest(dir, config, roots, NilVerificationObserver{}); err == nil {
+		if err := VerifyMptState(dir, config, roots, NilVerificationObserver{}); err == nil {
 			t.Errorf("Modified node should have been detected")
 		}
 	})
 }
+
+func TestVerification_CodeHashNotContainedInMptIsDetected(t *testing.T) {
```

Comment:
Please fix the name ... this test is not doing what the name indicates.

Please also check all other names. Consider that Mpt != MptState.